Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -57,6 +67,7 @@
 FETCH_SIZE = int(param.get('fetch_size', 50))
 INSECURE = not param.get('insecure', False)
 TIME_METHOD = param.get('time_method', 'Simple-Date')
+TIME_FORMAT = TIME_FORMATS_DICT.get(param.get('time_format', '%Y-%m-%d %H:%M:%S'))
```

Comment:
Why "invent the wheel"?
Let the customer set the format in a standard way.

Reply:
Sounds good!
But I thought cause the Elasticsearch format is different than the Python format.
See here
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html#built-in-date-formats

Reply:
Check out [dateutil](https://pypi.org/project/python-dateutil/), already in use in Content.
It will do ISO Format.